Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "LibGEOS"
 uuid = "a90b1aa1-3769-5649-ba7e-abc5a9d163eb"
 license = "MIT"
-version = "0.8.1"
+version = "0.8.2"
```

Comment:
@yeesian would it be possible to tag a release that includes this PR? That would be awesome, I already bumped the version.

Reply:
Thanks for the PR! I've added you as a contributor, so you can also tag releases :)

Reply:
Thanks a lot for adding me, I appreciate it!